src/0x.ts

```diff
@@ -300,17 +307,8 @@ export class ZeroEx {
                 const transactionReceipt = await this._web3Wrapper.getTransactionReceiptAsync(txHash);
                 if (!_.isNull(transactionReceipt)) {
                     intervalUtils.clearAsyncExcludingInterval(intervalId);
-                    const logsWithDecodedArgs = _.map(transactionReceipt.logs, (log: Web3.LogEntry) => {
-                        const decodedLog = this._abiDecoder.decodeLog(log);
-                        if (_.isUndefined(decodedLog)) {
-                            return log;
-                        }
-                        const logWithDecodedArgs: LogWithDecodedArgs = {
-                            ...log,
-                            ...decodedLog,
-                        };
-                        return logWithDecodedArgs;
-                    });
+                    const tryToDecodeLogOrNoOp = this._abiDecoder.tryToDecodeLogOrNoOp.bind(this._abiDecoder);
```

Comment:
Let's do `Noop` capitalization here. Although they used to be two works, it's now also `_.noop`.

Reply:
Let's not assign this method, instead in-line below.